rust-ceramic/src/help.md

```diff
@@ -0,0 +1,167 @@
+```
```

Comment:
I see we do have the help usage in the docs, great. Is this automated? Can we link to if from the places in the book that could benefit from it?

Reply:
Updated to generate a files that is included in the book.
CD would run the help commands and generate the help files that get included.
- `ceramic-one events slurp --help > src/ceramic-one-events-help.txt`
- `ceramic-one daemon --help > src/ceramic-one-daemon-help.txt`